### DISCUSSION TITLE
Is this a feature or a bug?

### DISCUSSION THREAD

**Discussion by royqh1979** [raysan5/raylib#2303] [Q&A]
In rmodel.c, DrawModelEx() function, we have the following line:

`model.transform = MatrixMultiply(model.transform, matTransform);`

So the model's transform matrix will get updated each time DrawModel/DrawModelEx is called.
Is it a feature, or a bug?

**Comment by raysan5** [comment]
This is the expected behaviour. Model transformation can be provided as a Matrix in `model.transform` but also as additional transform parameters to the function. Both are combined and applied to the Model before drawing.
